sensorhub-core/src/testFixtures/java/org/sensorhub/impl/datastore/AbstractTestDataStreamStore.java

```diff
@@ -596,30 +619,62 @@ public void testAddAndSelectByOutputName() throws Exception
         var ds4v0 = addSimpleDataStream(sysID3, "temp", TimeExtent.beginAt(now.plus(1, ChronoUnit.DAYS)));
         var ds5v0 = addSimpleDataStream(sysID3, "out1", TimeExtent.endNow(now.minus(60, ChronoUnit.DAYS)));
         dataStreamStore.commit();
-        
+
         // select from t0 to now
         DataStreamFilter filter = new DataStreamFilter.Builder()
-            .withOutputNames("out1")
-            .build();
+                .withOutputNames("out1")
+                .build();
         resultStream = dataStreamStore.selectEntries(filter);
-        
+
         testAddAndSelectByOutputName_ExpectedResults();
         checkSelectedEntries(resultStream, expectedResults, filter);
     }
-    
-    
+
+    @Test
+    @SuppressWarnings("unused")
+    public void testAddAndSelectByObservedProperty() throws Exception
+    {
+        Stream<Entry<DataStreamKey, IDataStreamInfo>> resultStream;
+
+        var now = Instant.now().truncatedTo(ChronoUnit.SECONDS);
+        var sysID1 = bigId(1);
+        var sysID2 = bigId(2);
+        var sysID3 = bigId(3);
+        var ds1v0 = addSimpleDataStreamWithDefinition(sysID1, "out1", TimeExtent.endNow(now.minus(365, ChronoUnit.DAYS)),"http://sensorml.com/ont/swe/property/Location");
```

Comment:
Maybe you could keep the definition in a variable so you can make sure the exact same is used in the filter